Style.css

```diff
@@ -1,43 +1,69 @@
-@import url('https://fonts.googleapis.com/css2?family=PT+Serif&display=swap');
-body {
-  margin: 0;
-  font-size: 12px;
-  line-height: 1.6;
-  width: 1200px;
+/* pt-serif-regular - latin */
+@font-face {
+  font-family: 'PT Serif';
+  font-style: normal;
+  font-weight: 400;
+  src: url('../fonts/pt-serif-v12-latin-regular.eot'); /* IE9 Compat Modes */
+  src: local(''),
+       url('../fonts/pt-serif-v12-latin-regular.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
```

Comment:
fix all the paths

```suggestion
       url('./fonts/pt-serif-v12-latin-regular.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
```
![Screenshot 2021-01-05 at 00 00 51](https://user-images.githubusercontent.com/10720827/103583920-495b5180-4ee9-11eb-915a-e91159571c24.png)